h2/stream.py

```diff
@@ -64,6 +64,13 @@ class StreamInputs(Enum):
     UPGRADE_SERVER = 18  # Added 2.3.0
 
 
+class StreamClosedBy(IntEnum):
```

Comment:
We don't need this to be an `IntEnum`. This doesn't ever need to be coerced to an integer for any reason, so we can just use an `Enum`. If you do that, the natural values of that enum are printable strings you can use in `recv_on_closed_stream`.